copied overveiw changes from earlier PR, included review feedback, an… (#395)
dgraph-io/dgraph-docs: content/dgraph-overview.md

[menu.main]
    name = "Dgraph Overview"
    identifier = "overview"
    weight = 2
+++
## Dgraph

Designed from day one to be distributed for scale and speed, **Dgraph** is the native Graph database with native GraphQL support. It is open-source, scalable, distributed, highly-available, and lightning fast.

Dgraph is different from other graph databases in a number of ways, including:

- **Distributed Scale**: &emsp; *Built from day 1 to be distributed, to handle larger data sets.*

- **GraphQL Support**: &emsp; *GraphQL is built in to make data access simple and standards-compliant. Unlike most GraphQL solutions, no resolvers are needed - Dgraph resolves queries automatically through graph navigation.*

- **True Free Open Source Software (FOSS)**: &emsp; *Dgraph is free to use, and [available on github](https://github.com/dgraph-io/dgraph).*

- **Fully Transactional and ACID Compliant**: &emsp; *Dgraph satisfies demanding OLTP workloads that require frequent inserts and updates.*

- **Text Search**: &emsp; *Full-text searching is included.*
## Who is using Dgraph & How

> Dgraph is a graph database with some special differentiators. Any graph database will be better when relationships and connections really matter, such as when querying across links in a social network, or detecting networks of interconnected fraudsters in a fraud application - these are natural graph use cases.

Reviewer: ```suggestion
Dgraph is a graph database with some special differentiators. Any graph database is better when relationships and connections really matter. For example, when querying across links in a social network, or detecting networks of interconnected fraudsters in a fraud application - these are natural graph use cases.
```

Reviewer: just copied back Rajie's edit
